lnrpc/lightning.proto

```diff
@@ -746,6 +746,10 @@ message GetTransactionsRequest {
 
     // An optional filter to only include transactions relevant to an account.
     string account = 3;
+
+    // A list of transaction hashes. When using REST, this field must be
+    // encoded as base64url.
+    repeated bytes tx_hash = 4;
```

Comment:
Also, it is default in our API that if a TXID is of type `bytes`, it is in the same byte order as it's coming out of the hash function. If it is `string`, then it's in the "human-readable" reverse byte order. So IMO we should use `chainHash, err := chainhash.NewFromStr(hash)` in the CLI, then set `req.TxHash[i] = chainHash[:]`. And a note here about the expected order also doesn't hurt.

Reply:
Yes, since it's a new field we can change the type without breaking backward compatibility. I think in this case `string` does make more sense.

Reply:
Thank you!